CoreScriptsRoot/CoreScripts/NotificationScript2.lua

```diff
@@ -175,11 +182,11 @@ PopupText.Text = "This is a popup"
 PopupText.Parent = PopupFrame
 
 --[[ Helper Functions ]]--
-local insertNotifcation = nil
+local insertNotification = nil
```

Comment:
I know you didn't write this code, but why do we need to forward declare these functions rather than declare then when we define them?